OPTIONS.md

```diff
@@ -0,0 +1,46 @@
+# VarTable Options
+
+## (Required)
+**--input-vcf**		input VCF file
+
+*	Do we need to differentiate between NGS_Mapper VCF (every position) versus lofreq VCF (only variants)?*
+
+**--input-bam**		input BAM file
+
+*	Do we assume BAM index file is in the same directory as BAM or should we include it as an option?
+
+*--input-bai	input BAM index file*
+
+**--input-ref**		input reference FASTA file
+
+*	Assuming we are using a GenBank ref, are we planning to parse annotated data?*
+
+*--accession	input reference fASTA accession number to parse GenBank information*
+
+**--minpercent**	input variant call percentage (at what frequency do we call a nt a variant)
+
+*	Required b/c we cannot assume the level 20%, 5%, 1%... it also prevents a mistaken run by user*
+
+**--minbq**		input minimum base nt quality
+
+*	Required b/c we cannot assume the quality 25%, 30%... it depends on --minpercent*
+
+
+## (Optional)
+**--input-prm**		input primer FASTA file
+
+**--mindepth**		input minimum depth of coverage for each nt base
+
+*	Should we set this automatically to a default of 10? For our usage, 10 does not change*
```

Comment:
We aren't doing any user defaults for any parameters like this. It causes problems when users make assumptions and it's not clear

Reply:
Same logic you're using with `--minpercent`